rsmtool/notebooks/comparison/score_distributions.ipynb

```diff
@@ -8,6 +8,41 @@
    ]
   },
   {
+   "cell_type": "code",
```

Comment:
I think it's great to have this, but I would argue it should be a separate notebook as I can imagine a case where you may want to exclude this from the report (e.g. nothing changed between the versions).